src/app/components/splash-screen/splash-screen.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit, OnDestroy , ViewEncapsulation } from '@angular/core';
+import { UtilityService } from 'src/app/services/utility.service';
+import {DEFAULT_IMAGE_TEXT} from "src/constants/constant";
+
+@Component({
+  selector: 'app-splash-screen',
+  templateUrl: './splash-screen.component.html',
+  styleUrls: ['./splash-screen.component.scss'],
+  encapsulation: ViewEncapsulation.None
+})
+export class SplashScreenComponent implements OnInit , OnDestroy {
+  showSplash: boolean;
+  imgText: string;
+  timeOutID: number;
+  iconImageUrl = "../../../assets/icons/logo.svg";
+
+  constructor(private _utilityService: UtilityService) {}
+
+  ngOnInit() {
+    this.imgText = DEFAULT_IMAGE_TEXT;
```

Comment:
You can directly assign the `DEFAULT_IMAGE_TEXT`. No need to do in on ngOnInit.

Reply:
Thank you vaia. I have changed accordingly.